example/index_cf.mjs

```diff
@@ -2,7 +2,7 @@ import CfClient from 'ff-nodejs-server-sdk';
 
 CfClient.init('1c100d25-4c3f-487b-b198-3b3d01df5794');
 
-setInterval(() => {
-    const value = CfClient.boolVariation('test', null, false);
+setInterval(async() => {
+    const value = await CfClient.boolVariation('test', null, false);
     console.log("Evaluation for flag test and target none: ", value);
 }, 10000);
```

Comment:
Don't worry about it for now, but it looks like prettier isn't running automatically on save for you. There's a mix of single and double quotes and a missing newline at the end of the file. Probably worth configuring prettier to run on save for `.mjs` files as well.